init.vim

```diff
@@ -177,13 +177,26 @@ set foldlevel=99
 
 
 if exists('g:vscode')
+    function SendLineWithoutIndent() range
+        let startLine = line("'<")
+        let indent = match(getline(startLine), '\S') + 1
+        let endLine = line("'>")
+        let currentLine = startLine
+        while currentLine <= endLine
+            call VSCodeCallRangePos("workbench.action.terminal.runSelectedText", currentLine, currentLine, indent, 9000, 0)<CR>
```

Comment:
Used `Call` instead of `Notify` because it was holding the calls until the end of the while loop, ending in calling the function many times with the latest `currentLine`